### PR TITLE
MI + VM template

### PR DESCRIPTION
**Pull request by milanbrkic-ms**

### CODE REVIEW

File: solutions/azure-sqlmi-v1/nested/vm.json [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+                        }
+                    }
+                ],
+                "networkSecurityGroup": {
+                    "id": "[variables('networkSecurityGroupId')]"
+                }
+            }
+        },
+        {
+            "type": "Microsoft.Compute/virtualMachines",
```
Comment (milanbrkic-ms) [inline solutions/azure-sqlmi-v1/nested/vm.json:125]: @unrealab could you please assist me in composing template for simple VM creation? I am getting following error with current config:
```
    "error": {
        "code": "BadRequest",
        "message": "Creation of VMs and Virtual Machine Scale Sets with ConfidentialVM security type is not supported in this region."
    }
```

Comment (unrealab) [reply solutions/azure-sqlmi-v1/nested/vm.json:125]: @milanbrkic-ms  - Confidential VMs are likely not available in all the regions, https://azure.microsoft.com/en-us/explore/global-infrastructure/products-by-region/?products=virtual-machines  this link has a table of what regions support Confidential VMs, You can look up by the Size of the VM you used. Ultimately, we'll want to restrict the possible regions the user can select based on the size of the VM you choose for the configuration. Let me know if you need more help.

**Review by unrealab — CHANGES_REQUESTED** [review]

File: solutions/azure-sqlmi-v1/nested/vm.json (hunk 1 above)
Comment (milanbrkic-ms) [reply solutions/azure-sqlmi-v1/nested/vm.json:110]: Got it!

File: solutions/azure-sqlmi-v1/azuredeploy-minimal.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                    },
+                    "administratorLoginPassword": {
+                        "value": "[parameters('administratorLoginPassword')]"
+                    },
+                    "requestedBackupStorageRedundancy": {
+                        "value": "Local"
+                    }
+                },
+                "templateLink": {
+                    "uri": "https://raw.githubusercontent.com/milanbrkic-ms/solution-center/iaas-mi/solutions/azure-sqlmi-v1/azuredeploy.json"
```
[the remark this answers is not in the document]
Comment (milanbrkic-ms) [reply solutions/azure-sqlmi-v1/azuredeploy-minimal.json]: Done

**Review by unrealab — APPROVED** [review]

**Pull request merged**